libantworld/renderer.cc

```diff
@@ -168,6 +203,30 @@ void Renderer::renderFirstPersonView(meter_t x, meter_t y, meter_t z,
     m_World.render();
 }
 //----------------------------------------------------------------------------
+void Renderer::renderFirstPersonView(meter_t x, meter_t y, meter_t z,
+                                     degree_t yaw, degree_t pitch, degree_t roll,
+                                     RenderTarget &renderTarget, bool bind, bool clear)
```

Comment:
I think we could do without this function and the one above, as the user could just call ``.bind()`` and ``.clear()`` manually -- but up to you.